app/src/main/javascript/bundles/experiment-page/html/index.html

```diff
@@ -126,8 +126,15 @@
             {"perplexity": 10 },{ "perplexity": 15 },{ "perplexity": 50 },{ "perplexity": 35 },{ "perplexity": 20 },{ "perplexity": 5 }],
             "umap": [{"n_neighbors": 5},{"n_neighbors": 100},{"n_neighbors": 50},{"n_neighbors": 10},{"n_neighbors": 30},{"n_neighbors": 15},{"n_neighbors": 3}]
           },
-            defaultPlotMethodAndParameterisation: {"tsne": {"perplexity": 50}, "umap": {"n_neighbors": 100}},
-            anatomogram:{},
+          defaultPlotMethodAndParameterisation: {"tsne": {"perplexity": 50}, "umap": {"n_neighbors": 100}},
+          anatomogram:{
+            "lung": [
+                "NCBITaxon_9606",
+                "PATO_0000461",
+                "UBERON_0002048",
+                "EFO_0000768"
+            ]
+          },
```

Comment:
what is the background of this piece of code?  It's not a review comment, but I would like to know the reason for that change. We can have a quick huddle?

Reply:
Sure, it's because I am trying to add some examples to test organ-anatomogram in this index.html demo.